test/unit/CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ if (NOT MSVC)
     SOURCES test-layout.cxx
     DEPENDS_ON gtest gtest_main ${CMAKE_THREAD_LIBS_INIT}
   )
+  raja_add_test(
+    NAME timer-test
+    SOURCES test-timer.cxx
+    DEPENDS_ON gtest gtest_main 
```

Comment:
Tests using GoogleTest need to depend on `${CMAKE_THREAD_LIBS_INIT}`, since it uses some pthread stuff.